examples/rpc_namespaces/src/main.rs

```diff
@@ -0,0 +1,74 @@
+//! Example of composing a did:ethr resolution server with multiple other namespaces/rpcs.
+//! This example creates a did:ethr resolver connected to the Sepolia Ethereum testnet and composes
+//! the rpc with the [`TurtleMethods`] rpc
+//!
+//! All did methods will be available under the `did_` namespace, all turtle methods under the `turtle_` namespace.
+//!
+//! Try running this example and executing the command `curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "rpc_methods"}' http://localhost:9999/` | jq .result
+//! (requires `curl` and `jq` packages to be installed)
+
+use std::str::FromStr;
+
+use anyhow::{Context, Result};
+use ethers::{
+    providers::{Provider, Ws},
+    types::Address,
+};
+use jsonrpsee::{server::Server, RpcModule};
+use lib_didethresolver::{rpc::DidRegistryMethods, DidRegistryServer, Resolver};
+use turtle_rpc::{TurtleMethods, TurtleRpcServer};
+
+mod turtle_rpc;
+
+/// The address of the DID Registry contract on the Ethereum Sepolia Testnet
+pub const DID_ETH_REGISTRY: &str = "0xd1D374DDE031075157fDb64536eF5cC13Ae75000";
```

Comment:
Is this deployed by us or is it more a open sourced contract maintained by a different team?

Reply:
it's the contract @jac18281828 worked jointly with VeramoLabs on